CHANGELOG.md

```diff
@@ -1,3 +1,91 @@
+Version 1.13.0
+==============
+
+This is a minor version release packed full of features and a general
+refactoring of how HoloViews renders widgets now built on top
+of the Panel library. Many thanks to the many contributors to this
+release either directly by submitting PRs or by reporting issues and
+making suggestions. Specifically we would like to thank @poplarShift,
+@jonmease, @flothesof, @julioasotodv, @ltalirz, @DancingQuanta,
+@ahuang, @kcpevey, @Jacob-Barkhak, @nluetts, @harmbuisman, @ceball,
+@mgsnuno, @srp3003, @jsignell as well as the maintainers @jbednar,
+@jlstevens and @philippjfr for contributing to this release.  This
+version includes the addition of a large number of features,
+enhancements and bug fixes:
+
+Major features:
+
+- Add `link_selection` to make linked brushing simple (#3951)
+- Completely replaced custom Javascript widgets with Panel based
+	widgets allowing for customizable layout (#84, #805)
+- Add `HSpan`, `VSpan`, `Slope`, `Segments` and `Rectangles` elements (#
+  3510, #3532, #4000)
+- Add Annotators to allow easily drawing, editing and annotating visual
+  elements (#1185)
+- Add a `Dataset.transform` method to easily perform data transforms
+  (#237, #3932)
+- Add support for cuDF GPU dataframes, cuPy backed xarrays and GPU
```

Comment:
Is it important to enforce a line-length limit in this file?  I'd favor letting things wrap in an editor so that they are organized semantically (e.g. with the PR number on the same line as its description) with no concern for line length.

Reply:
It is how we did things in this file in the past so I'm going to try to keep things consistent. I'll reformat again now I've turned the numbers into links.